composer.json

```diff
@@ -22,9 +22,10 @@
 		"issues": "https://github.com/kephson/paste-reference/issues"
 	},
 	"require": {
-		"php": "^8.1",
-		"typo3/cms-backend": "^12.4",
-		"typo3/cms-core": "^12.4"
+		"ext-pdo": "*",
+		"php": "^8.2",
+		"typo3/cms-backend": "^13.4 || dev.main",
```

Comment:
`dev.main` is wrong here.
It should be `dev-main` instead.

Personally I would leave it with `^13.4` only as this versions comes out in a week. Or change it to `^13.3 || dev-main` (_only during development, I would remove the `dev-main` constraint for the production release_) as with `^13.4 || dev-main` it isn't installable with the current sprint release 13.3 right?

BTW the **dev.main** leads to a failing pipeline at the moment.

Reply:
> ^13.4 || dev-main it isn't installable

mhm, not sure. Just `^13.3 || dev-main` should be fine for the moment, another adjustment in 1-2 weeks is no problem.